packages/react-components/react-textarea/src/stories/TextareaAppearance.stories.tsx

```diff
@@ -1,18 +1,34 @@
 import * as React from 'react';
-import { useId } from '@fluentui/react-utilities';
-import { Textarea } from '../index';
 import { Label } from '@fluentui/react-label';
 import { makeStyles, shorthands } from '@griffel/react';
+import { Text } from '@fluentui/react-text';
+import { Textarea } from '../index';
+import { useId } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
 
 const useStyles = makeStyles({
-  container: {
+  base: {
     display: 'flex',
     flexDirection: 'column',
-    ...shorthands.gap('10px'),
-    '& > div': { display: 'flex', flexDirection: 'column', ...shorthands.gap('10px'), ...shorthands.padding('10px') },
+    '& > div': {
```

Comment:
@behowell had some good feedback on my SpinButton PR to simplify the DOM and styles and get rid of the border radius.

This is how my changes ended up: https://github.com/microsoft/fluentui/pull/22995/files